consultation_analyser/pipeline/llm_summariser.py

```diff
@@ -7,9 +7,11 @@
 from consultation_analyser.consultations.models import Theme
 
 
-def dummy_generate_theme_summary(theme: Theme) -> str:
-    made_up_summary = (", ").join(theme.keywords)
-    return made_up_summary
+def dummy_generate_theme_summary(theme: Theme) -> dict[str, str]:
+    made_up_short_description = (", ").join(theme.keywords)
+    made_up_summary = f"Longer summary: {made_up_short_description}"
+    output = {"phrase": made_up_short_description, "summary": made_up_summary}
```

Comment:
Time to rename "phrase" to "short description" in here too?

Reply:
Yes, good idea - have updated.